app/_src/gateway/upgrade/dual-cluster.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Dual-cluster upgrade
+content_type: how-to
+purpose: Learn how to perform a dual-cluster upgrade for Kong Gateway
+---
+
+The dual-cluster upgrade strategy is a {{site.base_gateway}} upgrade option used primarily for traditional mode deployments and for control planes in hybrid mode. 
+
+With a dual-cluster upgrade, you deploy a new cluster of version Y alongside the current version X, so that two clusters serve requests concurrently during the upgrade process.
+
+![Dual-cluster upgrade workflow](/assets/images/products/gateway/upgrade/dual-cluster-upgrade.png)
+> _Figure 1: Dual-cluster upgrade workflow_
+
+This method has limitations on automatically generated runtime metrics that rely on the database. 
```

Comment:
Honestly, I'm not really sure either. It comes from the upgrade guide and I don't really understand what it refers to. 
@outsinre could you please clarify?

Reply:
@lena-larionova @cloudjumpercat 

Since two clusters with two databases. So some runtime metrics (like number of requests) are sent to two databases separately. Therefore, the metrics between the databases are not synced. If there is only one database, this is not an issue.